.travis.yml

```diff
@@ -10,4 +10,5 @@ rvm:
   - 2.6.3
 # Put this in your .travis.yml
 gemfile:
-  - gemfiles/rubocop_0.76.gemfile
+  - gemfiles/rubocop_0.77.gemfile
+  - gemfiles/rubocop_0.x.gemfile
```

Comment:
Is `gemfiles/rubocop_0.x.gemfile` necessary?

Reply:
- 0.77 for v0.77.x CI
- 0.x for v0.78 or above CI.

I created 0.x in case next rubocop version is released.

Reply:
Hm, it seems to me that  `gemfiles/rubocop_0.77.gemfile` and `gemfiles/rubocop_0.x.gemfile` redundant for RuboCop 0.77 testing. Is there any benefit to adding a gemfile for RuboCop 0.78 or higher? 
That hasn't been released yet. Probably it seems reasonable to have RuboCop 0.78 when RuboCop 0.78 will be released. WDYT?

Reply:
Yes, you're right, let's add another *.gemfile after next version is released. 

I've fixed this.

(I've created it since the existing appraise definition was `"~> 0.76"`, but I created one for rubocop v0.77.x, so I don't need it.)